apps/codeimage/src/client.d.ts

```diff
@@ -8,6 +8,8 @@ interface ImportMetaEnv {
   readonly VITE_ENABLE_MSW: boolean;
   readonly VITE_MOCK_AUTH: boolean;
   readonly VITE_API_BASE_URL: string | null;
+  readonly VITE_PRESET_LIMIT: number;
+  readonly VITE_PRESET_LIMIT_GUEST: number;
```

Comment:
We need to add these variable also in the CI in order to build the preview with the right params

https://github.com/riccardoperra/codeimage/blob/next/.github/workflows/main.yml#L191-L198

and

https://github.com/riccardoperra/codeimage/blob/d4cffd6141f79a24e404b4c9dbe33a8cd5bf7534/.github/workflows/prod-deploy.yml#L378-L387

Reply:
Facciamo in un'altra pr